op-batcher/batcher/driver.go

```diff
@@ -114,7 +115,8 @@ type BatchSubmitter struct {
 	txpoolState       TxPoolState
 	txpoolBlockedBlob bool
 
-	state *channelManager
+	state         *channelManager
+	prevCurrentL1 eth.L1BlockRef // cached CurrentL1 from the last syncStatus
```

Comment:
Wondering about pusing this `prevCurrentL1` variable inside `state`, so that `state` encapsulates all of the actual state. This would mean that it gets cleared when we decide to "clear state" however, which might not be what we want.

Reply:
I think it's better to keep it the way it currently is, because the `prevCurrentL1` matters in the context of `SyncStatus` handling, which happens at the level of the batcher main loop, not inside the channel manager. We should maybe instead rename the `state` variable just to `channelMgr` or so, because `state` is misleading 😅